system/api-server/src/file-upload/file-upload.controller.ts

```diff
@@ -81,7 +81,7 @@ export class ImageUploadController {
     @Res() response,
     @Query() params: GetUploadedFilesQuery,
   ) {
-    const { offset = 0, limit = 10 } = params;
+    const { offset = 0, limit = 10 } = params; 
```

Comment:
Can you explain what this does? Is it to paginate? 

If so, it doesn't look like the app is paginating correctly

Reply:
It was to paginate. But then I wasn't sure how to paginate on the frontend correctly so I left it out